qmk_cli/git.py

```diff
@@ -28,5 +28,5 @@ def clone(url, destination, branch):
         cli.log.error('git clone exited %d', p.returncode)
         return False
     else:
-        cli.log.info('Successfully cloned %s to %s!', cli.args.fork, cli.args.destination)
+        cli.log.info('Successfully cloned %s to %s!', url, destination)
         return True
```

Comment:
```suggestion
    return p.returncode
```
I'd be in favour of forwarding git's rc on errors with `exit(rc)`.